web-dev-server.config.js

```diff
@@ -2,7 +2,57 @@
 import { esbuildPlugin } from '@web/dev-server-esbuild';
 import fs from 'node:fs';
 
-const hasBaseParam = process.argv.includes('--base');
+const theme = process.argv.join(' ').match(/--theme=(\w+)/u)?.[1] ?? 'lumo';
```

Comment:
Let's update `yarn start:base` in package.json to use `--theme=base`.

Reply:
Done.